easybuild/easyconfigs/b/bwa-mem2/bwa-mem2-2.2.1-fix_legacy_code.patch

```diff
@@ -0,0 +1,72 @@
+diff -ru bwa-mem2-2.2.1.orig/src/ksort.h bwa-mem2-2.2.1/src/ksort.h
```

Comment:
@akesandgren I know the clue is in the filename, but please include at least a brief comment about the new patch.

Reply:
Done